test/unit_test/character_sheet_unit_tests/FocusUnitTests.java

```diff
@@ -0,0 +1,30 @@
+package unit_test.character_sheet_unit_tests;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import rpg_database.character_sheet.Focuses;
+import rpg_database.character_sheet.Focus;
+
+public class FocusUnitTests {
+
+	@Test
+	public void testGetFocus() {
+		Focus expected = new Focus(Focuses.COURAGE);
+		assertEquals(Focuses.COURAGE, expected.getFocus());
+	}
+
+	@Test
+	public void getFocuseValue() {
+		Focus expected = new Focus(Focuses.JUMPING);
+		assertEquals(2, expected.getFocuseValue());
+	}
+
+	@Test
+	public void testSetFocusImprovement() {
+		Focus expected = new Focus(Focuses.JUMPING);
+		expected.MakeFocusImproved();
+		assertEquals(true, expected.isFocusImproved());

```

Comment:
Itt sokkal pontosabb lenne azt tesztelni hogy ```assertEquals(3, expected.getValue());``` és egy külön tesztben megnézni hogy az isFocusImproved is jól működik -e ebben az esetben, mert így ebben a tesztben nem nézed meg hogy az értéken változtat -e a MakeFocusImproved, de cserébe a MakeFocusImproved mellett teszteled az isFocusImproved-ot is. Egyszerre csak egyet :) 

Illetve egy false esetet sem ártana nézni az isFocusImproved-ra.

Btw. lower camelcase-t használunk. "makeFocusImproved". Ez nem C#